bin/git-theta-filter

```diff
@@ -76,9 +76,9 @@ def clean(args):
     staged_file_contents = OrderedDict({"model_dir": theta_model_dir})
     # Sort the keys so we don't get changing diffs based on serialization order.
     for keys, param in sorted(utils.flatten(model_checkpoint).items()):
-         param_name = "/".join(keys)
-         tensor_metadata = OrderedDict({"shape": params.get_shape_str(param), "dtype": params.get_dtype_str(param), "hash": params.get_hash(param)})
-         staged_file_contents[param_name] = OrderedDict({"tensor_metadata": tensor_metadata}) 
+        param_name = "/".join(keys)
+        tensor_metadata = OrderedDict({"shape": params.get_shape_str(param), "dtype": params.get_dtype_str(param), "hash": params.get_hash(param)})
```

Comment:
This ordered dict isn't actually ensuring the order will be consistent.

The ordered dict records and will iterate based on insertion order, but the normal dict (which is created because arguments are evaluated before being passed) you are making as a parameter to the ordered dict init does not (pre python3.7). These when the orderdict iterates through the passed dict it will record that particular order which could be different between runs.

I think you need to manually make calls like `...["shape"] = ...` to ensure it will always be the same order.